package.json

```diff
@@ -1,56 +1,56 @@
 {
-	"name": "templating",
-	"version": "1.1.0",
-	"description": "eXist-db HTML Templating Library",
-	"homepage": "https://github.com/eXist-db/templating#readme",
-	"bugs": "https://github.com/eXist-db/templating/issues",
-	"keywords": [
-		"exist",
-		"exist-db",
-		"xml",
-		"xql",
-		"xquery"
-	],
-	"devDependencies": {
-		"@existdb/gulp-exist": "^4.3.2",
-		"@existdb/gulp-replace-tmpl": "^1.0.4",
-		"axios": "^0.21.1",
-		"chai": "^4.2.0",
-		"chai-xml": "^0.4.0",
-		"chokidar": "^3.5.3",
-		"delete": "^1.1.0",
-		"fs-extra": "^9.1.0",
-		"glob-stream": "^7.0.0",
-		"gulp": "^4.0.2",
-		"gulp-rename": "^2.0.0",
-		"gulp-zip": "^5.0.2",
-		"jsdom": "^16.4.0",
-		"mocha": "^10.1.0",
-		"xmldoc": "^1.1.2",
-		"yeoman-assert": "^3.1.1"
-	},
-	"author": {
-		"name": "The eXist-db Authors"
-	},
-	"license": "LGPL-2.1",
-	"scripts": {
-		"start": "npm install && npm run build",
-		"test": "gulp install:all && mocha test/mocha --recursive --exit && node test/xqs/xqSuite.js",
-		"test:watch": "mocha test/mocha --recursive --watch",
-		"build": "gulp build",
-		"build:all": "gulp build:all"
-	},
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/eXist-db/templating",
-		"license": "LGPL-2.1"
-	},
-	"funding": {
-		"type": "opencollective",
-		"url": "https://opencollective.com/existdb"
-	},
-	"overrides": {
-		"chokidar": "$chokidar",
-		"glob-stream": "$glob-stream"
-	}
+  "name": "templating",
+  "version": "1.2.0",
```

Comment:
Please do not update the version in a PR.
This should be done upon release only.

Reply:
Reverted and fixes. Changes made to `package.json` were removed from the commit.